lib/plugins/aws/package/compile/events/stream.js

```diff
@@ -32,6 +32,10 @@ class AwsCompileStreamEvents {
             maximumRecordAgeInSeconds: {
               anyOf: [{ const: -1 }, { type: 'integer', minimum: 60, maximum: 604800 }],
             },
+            functionResponseTypes: {
```

Comment:
Let's add that property to https://github.com/serverless/serverless/blob/master/docs/providers/aws/guide/serverless.yml.md as well